client/package.json

```diff
@@ -16,13 +16,18 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "babel-loader": "^8.1.0",
+    "dotenv": "^8.2.0",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-is": "^16.13.1",
+    "react-mailchimp-form": "^1.0.2",
+    "react-mailchimp-subscribe": "^2.1.0",
+    "react-owl-carousel": "^2.3.1",
```

Comment:
Can be removed

Reply:
The mailchimp subscribe is being used in the form, it is already connected with the Doiche Living MailChimp account. When it comes to the other two, yeah! I am going to delete them! thanks!